packages/hdwallet-walletconnect/src/walletconnect.test.ts

```diff
@@ -0,0 +1,77 @@
+import { WalletConnectWalletInfo, WalletConnectHDWallet } from "./walletconnect";
+import * as core from "@shapeshiftoss/hdwallet-core";
+
+describe("WalletConnectWalletInfo", () => {
+  const info = new WalletConnectWalletInfo();
+
+  it("vendor is WalletConnect", () => {
+    expect(info.getVendor()).toBe("WalletConnect");
+  })
+  it("hasOnDevicePinEntry returns false", () => {
+    expect(info.hasOnDevicePinEntry()).toBe(false);
+  })
+  it("hasOnDevicePassphrase returns false", () => {
+    expect(info.hasOnDevicePassphrase()).toBe(false);
+  })
+  it("hasOnDeviceDisplay returns false", () => {
+    expect(info.hasOnDeviceDisplay()).toBe(false);
+  })
+  it("hasOnDeviceRecovery returns false", () => {
+    expect(info.hasOnDeviceRecovery()).toBe(false);
+  })
+  it("hasNativeShapeShift returns false", () => {
+    expect(info.hasNativeShapeShift()).toBe(false);
+  })
+  it("supportsOfflineSigning returns false", () => {
+    expect(info.supportsOfflineSigning()).toBe(false);
```

Comment:
This should return true if `this.provider.wc.signTransaction(msg)` does not also broadcast the signed tx.